src/components/hostNav.jsx

```diff
@@ -27,7 +29,8 @@ export default function HostNav() {
       </Link>
       <Link
         className={`${
-          location.pathname === "/host/list"
+          location.pathname === "/host/list"||
+          location.pathname === `/host/list/${vanId.id}` ||location.pathname === `/host/list/${vanId.id}/price` || location.pathname === `/host/list/${vanId.id}/photo`
             ? "underline-offset-2 underline"
             : "text-[#4d4d4d]"
```

Comment:
You can use the NavLink component to achieve the same result rather than using the location object directly.